06LessonSix/in-class-project/css/intel-style.css

```diff
@@ -1,50 +1,43 @@
-
 body {
     font-family: 'Arial', sans-serif;
 }
-
 header, main, footer {
     width: 1024px;
     margin: auto;
 }
-
-!header {
-    background-color: #0071c5;
-    padding: 10px;

```

Comment:
You need to keep that 10px of padding. Please change and commit so I can bump your grade up to 100.